.github/workflows/nightly.yml

```diff
@@ -146,6 +356,10 @@ jobs:
       - name: Run Archetype for EE 8 Web Profile, SE 8, WildFly
         run: |
           mvn archetype:generate -DinteractiveMode=false -DaskForDefaultPropertyValues=false -DarchetypeGroupId=org.eclipse.starter -DarchetypeArtifactId=jakarta-starter -DarchetypeVersion=2.3.0-SNAPSHOT -DjakartaVersion=8 -Dprofile=web -DjavaVersion=8 -Druntime=wildfly -DoutputDirectory=app/wildfly -Dgoals="clean package"
+          if [ -f app/wildfly/jakartaee-hello-world/Dockerfile ]; then
```

Comment:
No need for this particular type of check. We should check for expected positive and negative cases.